sdk/src/wallet/account/operations/output_claiming.rs

```diff
@@ -266,7 +275,7 @@ where
                 // build new output with same amount, nft_id, immutable/feature blocks and native tokens, just
                 // updated address unlock conditions
 
-                let nft_output = if possible_additional_inputs.is_empty() {
+                let nft_output = if !enough_amount_for_basic_output {
                     // Only update address and nft id if we have no additional inputs which can provide the storage
                     // deposit for the remaining amount and possible NTs
```

Comment:
I don't understand this comment, we do same in both if/else?

Reply:
It's not `only` in this case, but `only` do this and don't update anything else